tools/rna_tools/cmsearch_deoverlap/cmsearch-deoverlap.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="cmsearch_deoverlap" name="CMsearch deoverlap" version="0.08+galaxy0" profile="21.05">
+    <description>removes lower scoring overlaps from cmsearch results</description>
+    <requirements>
+        <requirement type="package" version="5.26">perl</requirement>
+    </requirements>
+    <command><![CDATA[
+        ln -s '$clan_information' clan_info;
+        ln -s '$cmsearch_matches' cmsearch_matches;
+        LC_ALL=C perl '$__tool_directory__/cmsearch-deoverlap.pl' --clanin clan_info cmsearch_matches;
+        mv *.deoverlapped '$deoverlapped_matches'
+
+    ]]></command>
+    <inputs>
+        <param type="data" name="clan_information" format="txt" label="clan information" help="Each row in the file contains an RNA clan and its corresponding family members"/>
+        <param type="data" name="cmsearch_matches" format="txt" label="cmsearch output"/>
```

Comment:
cmsearch can have multiple outputs isn't it, do you need here to be more precise?

Reply:
according to the perl script this second param can conainer multiple cmsearch results. Is that correct?

Reply:
> according to the perl script this second param can conainer multiple cmsearch results. Is that correct?

Yes, you are right.

for example if you have the files "1.cmsearch.tblout", "2.cmsearch.tblout" and so on...

and the file "files_list" (containing the files names):
1.cmsearch.tblout
2.cmsearch.tblout
.
.
.

you can pass it as the files list in the command:
LC_ALL=C perl cmsearch-deoverlap.pl --clanin ribo.clan -l files_list

and the tool will generate an output for each file.
But I assume in this case the multiple dataset option in Galaxy can be used, right?

Reply:
Ah, so if you out N inputs, it creates N outputs?

Ok, then leave it like that. Nice!

Please change the help text, to singular. So not result__s__

Reply:
yes, exactly.